.github/actions/install-tackle/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Install tackle operator
```

Comment:
I do wonder if this should be two actions, one for installing the operator and another for creating the tackle CR.

Reply:
I think the right approach, in a future PR, would be to modify the CSV directly similar to how they do it in [OpenShift CI](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles) then rebuild the bundle + index and run that in minikube. That would give us the ability to test `tackle2-operator` with the same action.